applications/nrf_desktop/doc/power_manager.rst

```diff
@@ -1,7 +1,132 @@
++----------------------------------+------------------------+                                  +                      +------------------------------------+
+|                                  |                        |                                  |                      | :ref:`leds`                        |
++----------------------------------+------------------------+                                  +                      +------------------------------------+
+|                                  |                        |                                  |                      | :ref:`board`                       |
++----------------------------------+------------------------+                                  +----------------------+------------------------------------+
+| :ref:`nrf_desktop_main`          | ``module_state_event`` |                                  |                      |                                    |
++----------------------------------+------------------------+                                  +----------------------+------------------------------------+
+| :ref:`ble_state`                 | ``ble_peer_event``     |                                  |                      |                                    |
++----------------------------------+------------------------+                                  +----------------------+------------------------------------+
+| :ref:`usb_state`                 | ``usb_state_event``    |                                  |                      |                                    |
++----------------------------------+------------------------+                                  +----------------------+------------------------------------+
+| :ref:`hid_state`                 | ``hid_report_event``   |                                  |                      |                                    |
++----------------------------------+------------------------+                                  +----------------------+------------------------------------+
+| :ref:`nrf_desktop_power_manager` | ``power_down_event``   |                                  |                      |                                    |
++----------------------------------+------------------------+                                  +----------------------+------------------------------------+
+|                                  | ``wake_up_event``      |                                  |                      |                                    |
++----------------------------------+------------------------+----------------------------------+----------------------+------------------------------------+
+
+
+External dependencies
+*********************
+
+This module utilizes Zephyr :ref:`zephyr:power_management`.
+It depends on :option:`HAS_SYS_POWER_STATE_DEEP_SLEEP_1` being enabled and selects :option:`DEVICE_POWER_MANAGEMENT` and :option:`SYS_POWER_DEEP_SLEEP_STATES`.
```

Comment:
Kconfig options should have `CONFIG_` prefix

Reply:
Maybe we could merge `External dependencies` with the `Configuration` option. To make it consistent with other modules.

Reply:
Fixed